src/app/containers/Headings/helpers.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line no-useless-escape
+const regexPunctuationSymbols = /[\[\]\.,\/#?¿!$'"%^&*;:{}=\-_`~()؟؛٬«»！，。？、@#￥……（）：；《）《》“”〔〕’|]/gi;
+const regexSpaces = /\s+/g;
+
+const createId = (type, text) => {
```

Comment:
I'd probably name this something more meaningful, like `sanitiseString`, so that its name isn't limited to a single use case.